src/cephadm/cephadm

```diff
@@ -2292,10 +2292,34 @@ def command_version():
 @infer_image
 def command_pull():
     # type: () -> int
-    logger.info('Pulling latest %s...' % args.image)
-    call_throws([container_path, 'pull', args.image])
+
+    _pull_image(args.image)
     return command_inspect_image()
 
+
+def _pull_image(image) -> None:
+    logger.info('Pulling container image %s...' % image)
+
+    ignorelist = [
+        "error creating read-write layer with ID",
+        "net/http: TLS handshake timeout",
```

Comment:
this could be caused by network congestion .. should we retry? and if we do, maybe each iteration should exponentially back off...

Reply:
ok - let's not use the default retry count, which is ludicrously high with `10`